docs/tempo/website/_index.md

```diff
@@ -8,7 +8,7 @@ aliases:
 
 <p align="center"><img src="logo_and_name.png" alt="Tempo Logo"></p>
 
-Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki.
+Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Mimir, and Loki.
```

Comment:
I know that we're talking Tempo here, but why get rid of Prometheus? Can't folks use Tempo without Mimir?

Reply:
Is it more correct to say Grafana, Mimir, Prometheus, and Loki?

Reply:
honestly not sure

Reply:
Thank you for the explanation. If we're good with the text, I'll resolve this conversation. Learned stuff, too. :)